flask_ask/models.py

```diff
@@ -111,21 +111,33 @@ def render_response(self):
 
 class statement(_Response):
 
-    def __init__(self, speech):
-        super(statement, self).__init__(speech)
-        self._response['shouldEndSession'] = True
-
+    def __init__(self, speech=""):
+        if len(speech)!=0:
+            super(statement, self).__init__(speech)
+            self._response['shouldEndSession'] = True
+        else:
+            super(statement, self).__init__("Error: Please add text to statement message.");
```

Comment:
This way of handling an empty speech parameter isn't very ideal (especially for responses which represent the interaction between the program and the user).

If no speech is passed, the user will hear "Error: Please add text...", but the developer will not notice this problem unless they study the logging statements.

Instead of passing the error message to the response as speech, try raising an appropriate exception or logging the message as a warning or error so that the developer can quickly see the issue.